evaluation.py

```diff
@@ -229,18 +229,16 @@ def heatmaps_to_keypoints(self, heatmaps, threshold=HM_TO_KP_THRESHOLD):
 
             # Choose the max point in heatmap (we only pick 1 keypoint in each heatmap)
             # and get its coordinates and confidence
-            y, x = np.where(peaks == peaks.max())
+            y, x = np.unravel_index(np.argmax(peaks), peaks.shape)
 
-            if int(x[0]) > 0 and int(y[0]) > 0:
-                x_new = int(x[0])
-                y_new = int(y[0])
-                conf_new = peaks[y_new, x_new]
+            if peaks[y, x] > HM_TO_KP_THRESHOLD / 5:
```

Comment:
Why divide this by 5? Can we just change the constant?

Reply:
I wasn't sure if non-maximum suppression would reduce the max intensity so i picked an arbitrary number. anything below that threshold should have already been reset to 0 though